terrible.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import sys
+import os
+import time
+name='Yuki'
+class Greeter:
+    def greet(self):
+        print('Hello, world!')
+class NameGreeter(Greeter):
+    def __init__(self,who='world'):
+        self.__who=who
+    def greet(self):
+        print('Hello, {}!'.format(self.__who))
+if __name__=='__main__':
+    Greeter().greet()
+    NameGreeter().greet()
+    NameGreeter(name).greet()
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpcbzce6lr/terrible.py
+++ b/tmp/tmpcbzce6lr/terrible.py
@@ -11,7 +11,9 @@
         self.__who=who
     def greet(self):
         print('Hello, {}!'.format(self.__who))
-if __name__=='__main__':
+
+
+if __name__ == '__main__':
     Greeter().greet()
     NameGreeter().greet()
     NameGreeter(name).greet()
```